tools/hirondelle_crim_ogc_api_processes/crim_project_ogc_api_processes.xml

```diff
@@ -0,0 +1,224 @@
+				<option value="algae-usecase-workflow-copernicus-process">algae-usecase-workflow-copernicus-process: Processing on Sentinel-2 bands to evaluate algae bloom.</option>
+				<option value="algae-usecase-workflow-earth-search">algae-usecase-workflow-earth-search: Algae bloom for water quality assessment on Sentinel-2 L2A imagery offered by Earth-Search platform.</option>
+				<option value="algae-usecase-workflow-earth-search-process">algae-usecase-workflow-earth-search-process: Processing on Sentinel-2 L2A bands to evaluate algae bloom.</option>
+				<option value="calculate-band">calculate-band: Performs a calculation with bands.</option>
+				<option value="download-band-sentinel2-product-safe">download-band-sentinel2-product-safe: Downloads Copernicus products.</option>
+				<option value="download-band-sentinel2-stac-item">download-band-sentinel2-stac-item: Downloads Sentinel-2 GeoTiff from STAC Item.</option>
+				<option value="EchoProcess">EchoProcess</option>
+				<option value="file_index_selector">file_index_selector: File Index Selector</option>
+				<option value="file2string_array">file2string_array: File to String-Array</option>
+				<option value="jsonarray2netcdf">jsonarray2netcdf: JSON array to NetCDF</option>
+				<option value="metalink2netcdf">metalink2netcdf: Metalink to NetCDF</option>
+				<option value="ogc-tb16-land-cover-mapping">ogc-tb16-land-cover-mapping: OGC Testbed-16 Pixel-Wise Classifier for Land Cover Mapping</option>
+				<option value="plot-image">plot-image: Plots an image with colors.</option>
+				<option value="reproject-image">reproject-image: Performs image reprojection.</option>
+				<option value="select-products-sentinel2">select-products-sentinel2: Searches the specified catalog for Sentinel-2 products matching filtering criteria.</option>
+			</param>
+			<when value="algae-usecase-workflow-copernicus">
+				<param name="cookie" label="cookie" optional="false" value="" type="text"/>
+				<param name="date" label="date" optional="false" value="" help="No description provided!" type="text"/>
+				<param name="delta_Array_integer" label="delta" optional="false" value="" help="Please provide comma-seperated values of type integer here." type="text"/>
+				<param name="aoi" label="aoi" optional="false" value="" help="No description provided!" type="data" format="txt"/>
+				<param name="collection" label="collection" optional="false" value="" help="No description provided!" type="text"/>
+				<param name="cloud_cover_Array_float" label="cloud_cover" optional="false" value="" help="Please provide comma-seperated values of type float here." type="text"/>
+				<param name="s3_access_key" label="s3_access_key" optional="false" value="" help="Access key to Copernicus data provider. See https://documentation.dataspace.copernicus.eu/Registration.html and https://documentation.dataspace.copernicus.eu/APIs/S3.html#generate-secrets for details." type="text"/>
```

Comment:
we should not do that ... never safe credentials as tool parameters, those are captured forever for reproducibility and leak everywhere when you are assets.

Galaxy 24.1 that we will deploy this month, I hope, will have "Bring Your Own Storage", this will enable you to mount in all kind of S3 buckets into your workspace.